src/test/java/uk/gov/ons/ctp/response/action/endpoint/ActionRuleEndpointUnitTest.java

```diff
@@ -92,14 +82,34 @@ public void setUp() throws Exception {
     this.mockMvc =
         MockMvcBuilders.standaloneSetup(actionRuleEndpoint)
             .setHandlerExceptionResolvers(mockAdviceFor(RestExceptionHandler.class))
-            .setMessageConverters(new MappingJackson2HttpMessageConverter(new CustomObjectMapper()))
+            .setMessageConverters(new MappingJackson2HttpMessageConverter(objectMapper))
             .build();
 
-    actionRules = FixtureHelper.loadClassFixtures(ActionRule[].class);
     actionPlans = FixtureHelper.loadClassFixtures(ActionPlan[].class);
     actionTypes = FixtureHelper.loadClassFixtures(ActionType[].class);
   }
 
+  private ActionRulePostRequestDTO createActionRulePostRequestDTO(
```

Comment:
Not a fan of the private methods being above the tests or public methods. I feel like it takes the focus away from the tests

Reply:
Fixed